src/lib/types/restrictions.d.ts

```diff
@@ -3,23 +3,8 @@ declare namespace Restrictions {
   type TokenId = number
   type File = string
   type MetadataValue = File | string | TokenId | null
```

Comment:
If file is string why not `string | number | null`?

Reply:
I split it this way to match how it's typed on `dscp-node`, and in case we change the type of `File` or `TokenId` in the future